scripts/wakie.js

```diff
@@ -16,9 +16,9 @@ module.exports = function(robot) {
 
         // Send ping to two random victims in #general and react with random emoji
         var general = robot.adapter.client.rtm.dataStore.getChannelByName("general");
-        var members = general.members.filter(user => !user.deleted);
-        var victim1 = members[Math.floor(Math.random() * members.length)];
-        var victim2 = members[Math.floor(Math.random() * members.length)];
+        var members = general.members.filter(user => !user.deleted && !user.is_bot);
```

Comment:
I'm actually concerned that the reason it returns uqcsbot twice so frequently is because this list only contains 1 item, so filtering this out would leave an empty list of members.